editor/src/core/es-modules/package-manager/fetch-packages.ts

```diff
@@ -196,8 +196,9 @@ function failError(dependency: RequestedNpmDependency): DependencyFetchError {
 export async function fetchNodeModules(
   newDeps: Array<RequestedNpmDependency>,
   shouldRetry: boolean = true,
+  mode: 'canvas' | 'preview' = 'canvas',
```

Comment:
Please can you make this parameter non-optional

Reply:
fixed